st2actions/conf/syslog.conf

```diff
@@ -18,15 +18,11 @@ formatter=syslogVerboseFormatter
 args=()
 
 [handler_auditHandler]
-class=st2common.log.FormatNamedFileHandler
+class=st2common.log.ConfigurableSyslogHandler
 level=AUDIT
-formatter=jsonFormatter
-args=('logs/st2actionrunner.{pid}.audit.log',)
```

Comment:
Ta. I think that most people use the default log setup, so don't notice the
problem. I came across it when working on
https://github.com/StackStorm/showcase-ansible-chatops, as that was making
a change to use syslog.conf

On Fri, 15 Jul 2016 at 11:36 Lakshmi Kannan <notifications@github.com>
wrote:

> In st2actions/conf/syslog.conf
> <https://github.com/StackStorm/st2/pull/2774#discussion_r70902025>:
>
> >  level=AUDIT
> > -formatter=jsonFormatter
> > -args=('logs/st2actionrunner.{pid}.audit.log',)
>
> Ahh. Interesting. Ok, if this works, I am +1 to merge it :).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/StackStorm/st2/pull/2774/files/8b6b4968c5f0af64e9d4743590ff6b63e7eb25a9#r70902025>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADcKBp6lCJjBehUa7ji6fCJxJjM7Qm6mks5qVsf2gaJpZM4I8gJX>
> .
>